README.md

```diff
@@ -8,9 +8,7 @@ English | [简体中文](README-CN.md)
 
 ## Remind
 
-`github.com/taosdata/driver-go/v2` Complete refactoring of the v1 version and separate the built-in database operation
-interface `database/sql/driver` to the directory `taosSql` put other advanced functions such as subscription and stmt in
-the directory `af`
+`github.com/taosdata/driver-go/v2` Complete refactoring of the v1 version and separate the built-in database operation interface `database/sql/driver` to the directory `taosSql` put other advanced functions such as subscription and stmt in the directory `af`.
```

Comment:
My version:

`github.com/taosdata/driver-go/v2`  is completely refactoring based on the v1 version. It also separates the built-in database operation interface `database/sql/driver` into the directory `taosSql` and put other advanced functions such as subscription and stmt in the directory `af`.